src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup.Models/CommonModels/Utils.cs

```diff
@@ -25,6 +25,18 @@ public class PolicyConstants
     {
         public const int MaxAllowedRetentionDurationCount = 9999;
         public const int MaxAllowedDateInMonth = 28;
+
+        // day constants
+        public const int NumOfDaysInWeek = 7;
+        public const int NumOfDaysInMonth = 31;
+        public const int NumOfDaysInYear = 366;
```

Comment:
If these constants are already being used in service, then please ignore this comment.

Number of days in year doesn't match no. of weeks in year. Similarly other combinations don't match too. Is this intentional?